.travis.yml

```diff
@@ -2,10 +2,15 @@ language: node_js
 
 cache: yarn
 
+before_install:
+  - curl -o- -L https://yarnpkg.com/install.sh | bash
+  - export PATH="$HOME/.yarn/bin:$PATH"
+
 script:
   - commitlint-travis
   - yarn lint
   - yarn test
+  - BRANCH="${TRAVIS_PULL_REQUEST_BRANCH:-$TRAVIS_BRANCH}" && yarn canarist -r [https://github.com/untool/untool.git -b $BRANCH -c] -r https://github.com/xing/hops.git
```

Comment:
why not `npm run …`?

and same concern as in hops: I think it makes sense to have the command available outside of the travis context.